Tools/WebKitBot/src/WebKitBot.mjs

```diff
@@ -471,7 +472,9 @@ Type \`help COMMAND\` for help on my individual commands.`,
         let results;
         try {
             const webkitPatchPath = path.resolve("BotWebKit", "Tools", "Scripts", "webkit-patch");
-            results = await execFileAsync(webkitPatchPath, [
+            var pythonPath = which.sync('python3')
+            results = await execFileAsync(pythonPath, [
```

Comment:
What's the reason to invoke like this, rather than invoking it via `#!/usr/bin/env python3`? Presumably we're finding the same `python3` interpreter either way, as we'll have the same `PATH` either way, no?

Reply:
Oh, `sys.executable`. That should absolutely, definitely, be populated the exact same either way?

Reply:
sys.executable is not getting populated when BotWebKit is invoked directly using execFileAsync.

Reply:
Right. Also, setting `argv0` in options doesn't work, even though it definitely seems like it should.

Python appears to be doing some illogical things here.

Reply:
I wonder why, I wonder what `execFileAsync` is doing such that CPython is unable to determine the path of the interpreter…

Reply:
I doubt that it could do much interesting on Node.js side. This general kind of problem appears to be common with Python, see https://bugs.webkit.org/show_bug.cgi?id=225172 for an earlier instance unrelated to Node.